src/asciidoc/docs/guides-general/capture-user-input.adoc

```diff
@@ -0,0 +1,139 @@
+include::{includespath}/steps-codeeditor.adoc[tag=terminal-restart-process]
+====
+
+* Import @withkoji/core in your front-end code.
++
+[source,javascript]
+----
+import Koji from '@withkoji/core';
+----
+
+* Call the appropriate `Koji.ui.capture` control.
++
+[source,javascript]
+----
+myVariable = await Koji.ui.capture.<control-name>( <control-parameters> )
+----
++
+** `.audio` – Prompts the user to select or upload an audio file.
+<<core-frontend-ui-capture#audio,API reference>>
+
+** `.color` – Prompts the user to select or specify a color.
+<<core-frontend-ui-capture#color,API reference>>
++
+If `allowAlpha : true` is included in the options, the *Opacity* slider is visible in the color selector modal.
+
+** `.custom` – Prompts the user to interact with a custom control.
+<<core-frontend-ui-capture#custom,API reference>> +
+Learn more: <<build-custom-controls#>>
+
+** `.file` – Prompts the user to upload a file of any type.
+<<core-frontend-ui-capture#file,API reference>>
+
+** `.image` – Prompts the user to select or upload an image file.
+<<core-frontend-ui-capture#image,API reference>>
+
+** `.link` – Prompts the user to enter an external URL.
+<<core-frontend-ui-capture#link,API reference>>
+
+** `.media` – Prompts the user to select, specify, or upload an image, a file, an audio, or a video.
+<<core-frontend-ui-capture#media,API reference>>
+
+** `.range` – Prompts the user to select a value from a numeric range.
+<<core-frontend-ui-capture#range,API reference>>
+
+** `.select` – Prompts the user to select from a predefined list of options.
+<<core-frontend-ui-capture#select,API reference>>
++
+If the list has 1 to 4 options, the options are presented as radio buttons. +
+If the list has 5 or more options, the list is presented as a dropdown.
+
+** `.video` – Prompts the user to select or upload a video file.
+<<core-frontend-ui-capture#video,API reference>>
+
+== Controls for various file types
+
+The @withkoji/core package provides multiple APIs for selecting or uploading files.
+If your app needs a specific type of file (audio, image, or video), use the specific API for that file type if possible.
+That is, instead of `.media` or `.file`, use the following:
+
+* To accept only audio files, use `.audio`.
+* To accept only images, use `.image`.
+* To accept only videos, use `.video`.
+
+The specific APIs:
+
+* Filter the selections offered to the user. Examples:
+** The popup menu for `.audio` includes "Capture audio" and "Browse Sound Packs".
+** The popup menu for `.image` includes "Take photo" and "Browse Image Packs".
+* Restrict the file formats that can be uploaded. Examples:
+** The file selector for `.audio` automatically filters for .mp3, .wave, and other standard audio file formats.
+** The file selector for `.image` automatically filters for .png, .jpg, and other standard image file formats.
+* Allow file-type-specific actions, such as automatically editing images and watermarking videos.
```

Comment:
"file-type specific actions" - This phrase makes it sound like the user is taking an action, which isn’t accurate. Rather, the app can apply changes to files during upload. That’s a benefit of using the audio, image, video, or media upload methods.

Reply:
BEFORE: "Allow file-type-specific actions, ..."
AFTER: "Apply file-type-specific changes to the uploaded file, ..."